nncf/quantization/algorithms/fast_bias_correction/ov_backend.py

```diff
@@ -0,0 +1,165 @@
+        skip_metatypes = ['Convert']
+        needed_bias_type = 'Constant'
+
+        potential_bias_queue = deque([n.node for n in bias_node.input_values()])
+        while potential_bias_queue:
+            potential_bias = potential_bias_queue.popleft()
+            if potential_bias.get_type_name() in skip_metatypes:
+                # We goes thorough 0 port in assumption that bias graph without branching
+                potential_bias_queue.append(potential_bias.input_value(0).node)
+            elif potential_bias.get_type_name() == needed_bias_type:
+                bias_value = potential_bias.get_data()
+                return bias_value.flatten(), bias_value.shape
+            continue
+
+        raise RuntimeError('Could not find the bias value of the node')
+
+    @staticmethod
+    def get_activation_port_ids_for_bias_node(model: ov.Model, biased_node: NNCFNode) -> Tuple[int, int]:
+        return 0, 0
+
+    @staticmethod
+    def is_quantized_weights(biased_node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        weight_port_id = biased_node.layer_attributes.weight_port_id
+        weight_node = nncf_graph.get_input_edges(biased_node)[weight_port_id].from_node
+        fq_type = OV_OPERATION_METATYPES.get_operator_metatype_by_op_name('FakeQuantize')
+        return weight_node.metatype == fq_type
+
+    @staticmethod
+    def get_bias_port_id(bias_node: NNCFNode) -> int:
+        return bias_node.layer_attributes.weight_port_id
+
+    @staticmethod
+    def process_model_output(raw_data: Dict, output_name: str) -> OVNNCFTensor:
+        return OVNNCFTensor(raw_data[output_name])
+
+    @staticmethod
+    def is_node_with_bias(biased_node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        bias_node = OVFastBiasCorrectionAlgoBackend.get_bias_node(biased_node, nncf_graph)
+        if bias_node is None:
+            return False
+
+        needed_bias_type = OV_OPERATION_METATYPES.get_operator_metatype_by_op_name('Constant')
+        skip_metatypes = [OV_OPERATION_METATYPES.get_operator_metatype_by_op_name('Convert')]
+
+        potential_bias_queue = deque(nncf_graph.get_previous_nodes(bias_node))
+        while potential_bias_queue:
+            potential_bias = potential_bias_queue.popleft()
+            if potential_bias.metatype in skip_metatypes:
+                # We goes thorough 0 port in assumption that bias graph without branching
+                potential_bias_queue.append(nncf_graph.get_previous_nodes(potential_bias)[0])
+            elif potential_bias.metatype == needed_bias_type:
+                return True
+            continue
+        return False
+
+    @staticmethod
+    def get_bias_node(biased_node: NNCFNode, nncf_graph: NNCFGraph) -> Optional[NNCFNode]:
+        add_bias_node = nncf_graph.get_next_nodes(biased_node)[0]
+        if add_bias_node.metatype == OV_OPERATION_METATYPES.get_operator_metatype_by_op_name('Add'):
+            return add_bias_node
+        return None
```

Comment:
I think this method is not required here. Could we remove it?

Reply:
Why it's not required?